### PR TITLE
globally defined colors

### LINKED ISSUE #9: Color definition
As a developer, I don't want to type hex values to define the same color in different places. Therefore we should implement fixed colors, like here:

https://swairaq.medium.com/custom-colors-in-react-native-53f3591961cc

How should we name those colors.
Let's don't make a science out of it but for naming structure, here are some inspirations:
https://m2.material.io/develop/ios/components/theming/color
https://hackernoon.com/interface-colors-from-palette-to-themes

Acceptance criteria:
Globally defined colors with meaningful and intuitive names!

### PR DESCRIPTION
**Pull request by Ashz22**
Closes #9

### CODE REVIEW

File: app/utils/Color.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { on } from "events";
```
Comment (cma5) [inline app/utils/Color.ts]: Do you need this import? If not, just delete it.

**Review by cma5 — CHANGES_REQUESTED** [review]

**Comment by cma5** [comment]
Otherwise it looks good :)

File: app/utils/Color.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const colors =
```
Comment (cma5) [inline app/utils/Color.ts]: Always write classes as nouns with in PascalCase (Capital Letters)

File: app/utils/Color.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    tagPurple: '#BD82D9',
+    onTagRed: '#A13B3B',
+    onTagOrange: '#8A6926',
+    onTagYellow: '#7E7630',
+    onTagGreen: '#266940',
+    onTagBlue: '#32578E',
+    onTagPurple: '#70398A',
+};
+
+export default colors;
```
Comment (cma5) [inline app/utils/Color.ts]: also change it here

**Review by cma5 — CHANGES_REQUESTED** [review]

File: app/utils/Color.ts (hunk 2 above)
Comment (theopfr) [reply app/utils/Color.ts]: Hm, this is no class though, is it? It's a normal variable so I'd say it should remain lowercase? (debatable but not very important)

Comment (cma5) [reply app/utils/Color.ts]: I know that it holds only variables but by definition it's a class or am I missing sth. Anyway...

**Review by cma5 — APPROVED** [review]

**Pull request merged**